packages/server/src/app/db/notifications/notifications.module.ts

```diff
@@ -0,0 +1,38 @@
+import { Module } from '@nestjs/common';
+import { MongooseModule } from '@nestjs/mongoose';
+import { NotificationSchema } from './notifications.schema';
+import { UnpublishedNotificationSchema } from './unpublished-notifications.schema';
+import { NotificationsService } from './notifications.service';
+import { UnpublishedNotificationsService } from './unpublished-notifications.service';
+import { NotificationSubscriptionSchema } from './notification-subscriptions.schema';
+
+@Module({
+  imports: [
+    MongooseModule.forFeatureAsync([
+      {
+        name: 'Notification',
+        useFactory: () => {
+          const schema = NotificationSchema;
+          return schema;
+        }
+      },
+      {
+        name: 'UnpublishedNotification',
+        useFactory: () => {
+          const schema = UnpublishedNotificationSchema;
+          return schema;
+        }
+      },
+      {
+        name: 'NotificationSubscription',
+        useFactory: () => {
+          const schema = NotificationSubscriptionSchema;
+          return schema;
+        }
+      }
+    ])
+  ],  
+  providers: [NotificationsService, UnpublishedNotificationsService],
+  exports: [NotificationsService],
```

Comment:
`UnpublishedNotificationService` not being exported is intentional--it's an implementation detail. I may even fold its functionality into the `NotificationService` itself, I'm not sure yet.